accessible/fieldview.component.js

```diff
@@ -47,27 +47,21 @@ blocklyApp.FieldView = ng.core
       <label #label id="{{treeService.createId(label)}}">
         {{field.getText()}}
       </label>
-      {{setLabelledBy(listItem, concatStringWithSpaces("blockly-argument-text", label.id))}}
+      {{utilsService.setLabelledBy(listItem, utilsService.concatStringWithSpaces("blockly-argument-text", label.id))}}
     </li>
     `,
     inputs: ['field', 'level', 'index', 'parentId'],
   })
   .Class({
-    constructor: [blocklyApp.TreeService, function(_treeService) {
+    constructor: [blocklyApp.TreeService, blocklyApp.UtilsService, 
+        function(_treeService, _utilsService) {
       this.optionText = {
         keys: []
       };
       this.text = '';
       this.treeService = _treeService;
+      this.utilsService = _utilsService;
```

Comment:
Try not to spread utilsService all over the template. If you have to use it (and I still think it is better if you can find a way to set the labelled-by attribute directly), define a single method on the class that calls it, and have the template use that method. E.g.:

    setLabelledByAttr(item, prefix, labelId)

or something like that. The same applies to the other files.